server/models/User.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcryptjs');
+// const crypto = require('crypto');
+
+const userSchema = new mongoose.Schema({
+	username: String,
+	password: String,
+	firstName: String,
+	lastName: String,
+	time: Date
```

Comment:
What does this field contain? Remember that I, and everyone else, is lazy and prefers to not look at documentation

Reply:
Which field? Time I think would ideally be when the animal was seen, but is currently set up to be when the Sighting is added to the db

Reply:
`time`.
This is the user model, so idk how that would be the time animal was seen at.
`spottedAt` or `sightedAt` would be clearer if it were on the sighting model. Maybe it should be `registeredAt` here?

Reply:
d'oh. Yes we can change to registeredAt as this is added when the passport signup runs